src/components/pages/Home.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { fetchCountries } from '../../redux/country';
+import CountryList from '../CountryList';
+import { getMapUrl } from '../mapUtils';
+
+const Home = () => {
+  const worldImage = getMapUrl('world');
+  const dispatch = useDispatch();
+  const [searchTerm, setSearchTerm] = useState('');
+
+  const upDateSearch = (e) => {
+    setSearchTerm(e.target.value);
+  };
+
+  useEffect(() => {
+    dispatch(fetchCountries());
+  }, []);
+
+  return (
+    <>
+      <div
+        className="container mx-auto h-40 opacity-10"
+        style={{
+          backgroundImage: `url(${worldImage})`,
+          backgroundSize: 'cover',
+          backgroundRepeat: 'no-repeat',
+          backgroundPosition: 'center',
+        }}
+      />
```

Comment:
- [ ] [OPTIONAL] For a better User Interface and Experience, let's ensure the map of each region displayed on the details page fits properly into the div/container.
### Your design for the home page taking Colombia as an example
![Screenshot from 2022-05-20 23-59-24](https://user-images.githubusercontent.com/58692590/169623626-13cb6358-4612-444a-8e9a-bdde3c869e6a.png)

### Your design for the details page taking Colombia as an example
![Screenshot from 2022-05-21 00-18-00](https://user-images.githubusercontent.com/58692590/169623765-f95e9812-4e9e-4cec-8d05-139e02051927.png)